### PR TITLE
Fixes/pickers x icon

### PR DESCRIPTION
**Pull request by joneff**
Fixes position of [x] clear icons in combo, autocomplete and multiselect.

### CODE REVIEW

**Comment by inikolova** [comment]
Clear icon overlaps the text

**Comment by inikolova** [comment]
Loading indicator in AutoComplete is not vertically aligned

**Review by magdalenaan — COMMENTED** [review]
x-icon in multiselect is upper https://www.screencast.com/t/hBqTTb82F7o

**Comment by inikolova** [comment]
In MultiSelect for jQuery the **k-i-loading** icon is always visible

**Comment by inikolova** [comment]
In RTL mode and filtering enabled, the DropDownList appearance is broken

**Review by gyoshev — CHANGES_REQUESTED** [review]
The code appears to be clearer, however the above comments need to be addressed before merging.

| Before | After |
|-|-|
| image | image |
| image | image |
| image | image |
| image | image <br> (also gets 100% width) |

Also, the clear button is now always visible in the ComboBox and MultiSelect -- is this by design?

File: scss/common/_base.scss [hunk 1]
```diff
@@ -540,7 +540,7 @@
         content: "";
         box-sizing: inherit;
         border-radius: 50%;
-        border-width: .03em;
+        border-width: .03125em;
```
Comment (joneff) [inline scss/common/_base.scss]: This value should be 0.05 as per https://github.com/telerik/kendo-theme-default/pull/768

**Comment by gyoshev** [comment]
PR moved to monorepo.

**Pull request closed without merging**